### PR TITLE
chore: Add support for Python 3.10

### PR DESCRIPTION
**Pull request by svillegas-cdd**
* Add Python 3.10 to Setuptools script.
* Add Tox environment for Python 3.10.
* Add Python 3.10 to CircleCI configuration.
* Add Python 3.10 .black target-version

More information: https://docs.python.org/3.10/whatsnew/3.10.html

Ref: https://cordada.aha.io/features/COMPCLDATA-161

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -150,6 +150,7 @@ workflows:
                 - "3.7.9"
                 - "3.8.13"
                 - "3.9.1"
+                - "3.10.6"
```
Comment (ycouce-cdd) [inline .circleci/config.yml:153]: I think we should use the final available version 3.10.9

```suggestion
                - "3.10.9"
```

Ref: https://docs.python.org/3.10/whatsnew/changelog.html#python-3-10-9-final

**Review by ycouce-cdd — COMMENTED** [review]
Please, add a reference to the new features included in this new version of Python, e.g.

> "More information: https://docs.python.org/3.10/whatsnew/3.10.html"

Both in the commit message and in the description of the PR.

On the other hand, we should also update the version of Python used in the CI jobs, will we do it in this PR @jtrobles-cdd ?

https://github.com/fyntex/lib-cl-sii-python/blob/e07eddc79ea03839421e9d4dcfbaddcc2d372104/.circleci/config.yml#L87

https://github.com/fyntex/lib-cl-sii-python/blob/e07eddc79ea03839421e9d4dcfbaddcc2d372104/.circleci/config.yml#L121

File: .circleci/config.yml (hunk 1 above)
Comment (fpinto-cdd) [inline .circleci/config.yml:153]: Here we should test for version 3.10.9 since it is the latest release

**Review by fpinto-cdd — CHANGES_REQUESTED** [review]

**Review by ycouce-cdd — APPROVED** [review]

**Review by fpinto-cdd — APPROVED** [review]

**Review by jtrobles-cdd — COMMENTED** [review]
:heavy_check_mark:

**Pull request merged**